htlcswitch/switch.go

```diff
@@ -1164,6 +1164,10 @@ func (s *Switch) handlePacketForward(packet *htlcPacket) error {
 			return err
 		}
 
+		if circuit == nil {
```

Comment:
could use a comment here explaining when we expect the circuit to be nil

Reply:
done, could you check out my comment and see if it's all good

Reply:
looks good to me 👍